web/ensign-user-ui/src/config/default.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Here's a link to the config package documentation: https://github.com/node-config/node-config

This is a basic configuration for now that I'm sure will change.